packages/ui-drilldown/src/Drilldown/README.md

```diff
@@ -181,6 +514,242 @@ const contactsData = [
+
+This option will always display on the page when needed and cannot be disabled.
+
+```js
+---
+example: true
+render: false
+---
+class GroupsExample extends React.Component {
+  state = {
+    showTitle: true,
+    showAction: true,
+    showAlternativeBackLabel: false
+  }
+
+  render() {
+    const { showTitle, showAction, showAlternativeBackLabel } = this.state
+
+    return (
+      <Flex alignItems='start'>
+        <Flex.Item>
+          <Drilldown rootPageId='root' width="20rem" maxHeight="30rem">
+            <Drilldown.Page
+              id="root"
+              renderTitle={showTitle && 'Root page'}
+              renderActionLabel={showAction && 'Action!'}
+            >
+              <Drilldown.Option id="option1" subPageId='secondPage'>
+                Option
+              </Drilldown.Option>
+              <Drilldown.Option id="option2" subPageId='secondPage'>
+                Option
+              </Drilldown.Option>
+              <Drilldown.Option id="option3" subPageId='secondPage'>
+                Option
+              </Drilldown.Option>
+            </Drilldown.Page>
+
+            <Drilldown.Page
+              id="secondPage"
+              renderTitle={showTitle && 'Second page'}
+              renderActionLabel={showAction && 'Action!'}
+              renderBackButtonLabel={showAlternativeBackLabel
+                ? ((prevPageTitle) => prevPageTitle ? `Back to ${prevPageTitle}` : 'Back')
+                : undefined}
+            >
+              <Drilldown.Option id="option1">Option</Drilldown.Option>
+              <Drilldown.Option id="option2">Option</Drilldown.Option>
+              <Drilldown.Option id="option3">Option</Drilldown.Option>
+            </Drilldown.Page>
+          </Drilldown>
+        </Flex.Item>
+
+        <Flex.Item padding="0 0 0 large">
+          <FormFieldGroup description="Drilldown Group example settings">
+            <Checkbox
+              checked={this.state.showTitle}
+              label="Show page title"
+              variant="toggle"
+              onChange={
+                () => {
+                  this.setState({ showTitle: !this.state.showTitle })
+                }
+              }
+            />
+            <Checkbox
+              checked={this.state.showAction}
+              label="Show action label"
+              variant="toggle"
+              onChange={
+                () => {
+                  this.setState({ showAction: !this.state.showAction })
+                }
+              }
+            />
+            <Checkbox
+              checked={this.state.showAlternativeBackLabel}
+              label="Show alternative back navigation label"
+              variant="toggle"
+              onChange={
+                () => {
+                  this.setState({ showAlternativeBackLabel: !this.state.showAlternativeBackLabel })
```

Comment:
this does not work when all the other switches are off

Reply:
Well yes, because if the titles are turned off, there is no title to display in the Back navigation. I modified the checkbox label to make it more obvious.

Reply:
Also added some logic that turns it off and disables it when the titles are turned off.